tools/pythonpkg/tests/fast/pandas/test_pandas_df_none.py

```diff
@@ -6,14 +6,10 @@
 
 
 class TestPandasDFNone(object):
+    # This used to decrease the ref count of None
     def test_none_deref(self):
         con = duckdb.connect()
         gc.collect()
-        none_refcount = sys.getrefcount(None)
         for _ in range(1000):
             df = con.sql("select NULL::VARCHAR as a").df()
```

Comment:
Instead of looping in python, we can probably return a million (or more) `range()` result which immediately crashes the interpreter if you want to verify that ;-)
`db.sql('select null::text from range(1000000)').df()`